src/main/java/com/taskone/TaskOneAtomic.java

```diff
@@ -0,0 +1,65 @@
+package com.taskone;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.IntStream;
+
+public class TaskOneAtomic {
+    static AtomicInteger value = new AtomicInteger(0);
+    public int iterations = 10;
+
+    public void add(int delta) {
+        value.addAndGet(delta);
+    }
+
+    Thread addOne = new Thread("Add One") {
+        public void run() {
+            for (int i = 0; i <= iterations; i++) {
+                System.out.println("AddOne thread started");
+                add(1);
```

Comment:
inlinde value.addAndGet(delta), bad practice to extract one line method, which is simple like that.